### PR TITLE
security policy

### PR DESCRIPTION
**Pull request by Eomm**
#### Checklist

- [ ] run `npm run test` and `npm run benchmark`
- [ ] tests and/or benchmarks are included
- [x] documentation is changed or added
- [x] commit message and code follows [Code of conduct](https://github.com/fastify/fastify/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Security Policy
+
+This document describes the management of vulnerabilities for the Fastify project.
+
+
+## Reporting vulnerabilities
+
+Individuals who find potential vulnerabilities in Fastify are invited to complete a vulnerability report on the dedicated HackerOne organization: [https://hackerone.com/nodejs-ecosystem](https://hackerone.com/nodejs-ecosystem) for the modules of Node.js
```
Comment (jsumners) [inline SECURITY.md]:
```suggestion
Individuals who find potential vulnerabilities in Fastify are invited to complete a vulnerability report via the dedicated HackerOne tool for Node.js modules: [https://hackerone.com/nodejs-ecosystem](https://hackerone.com/nodejs-ecosystem).
```

File: SECURITY.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+### How to report a vulnerabiliy
+
+It is of the outmost importance that you read carefully [**HOW TO REPORT A VULNERABILIY**](https://github.com/nodejs/security-wg/blob/master/processes/third_party_vuln_process.md) written by the Security Working Group of Node.js.
```
Comment (jsumners) [inline SECURITY.md]:
```suggestion
It is of the utmost importance that you read carefully [**HOW TO REPORT A VULNERABILIY**](https://github.com/nodejs/security-wg/blob/master/processes/third_party_vuln_process.md) written by the Security Working Group of Node.js.
```

File: SECURITY.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+
+## Handling vulnerability reports
+
+When a potential vulnerability is reported and confirmed the Fastify Core Team will intervene in the
+`follow-up` stage of the process following the workflow and the timings described in the Security WG's document.
+
+### Vulnerabilities found outside this process
+
+⚠ Contrarily to the Security WG's document, the Fastify project does not support any reporting outside the HackerOne process.
```
Comment (jsumners) [inline SECURITY.md]:
```suggestion
⚠ The Fastify project does not support any reporting outside the HackerOne process.
```

File: SECURITY.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+
+## The Fastify Core team
+
+They are responsible for the management of this process.
```
Comment (jsumners) [inline SECURITY.md]:
```suggestion
The core team is responsible for the management of this process.
```

File: SECURITY.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+`follow-up` stage of the process following the workflow and the timings described in the Security WG's document.
+
+### Vulnerabilities found outside this process
+
+⚠ The Fastify project does not support any reporting outside the HackerOne process.
+
+
+## The Fastify Core team
+
+The core team is responsible for the management of this process.
```
Comment (mcollina) [inline SECURITY.md]: This it would be good to be a link.

**Comment by mcollina** [comment]
I've been managing this all alone for the vast majority of cases.

I think we should note that the above applies to all the plugins in the org as well?

**Pull request merged**